docs/docsite/rst/user_guide/playbooks_module_defaults.rst

```diff
@@ -120,8 +132,12 @@ In a playbook, you can set module defaults for whole groups of modules, such as
         group/aws:
           region: us-west-2
       tasks:
-      - aws_s3_bucket_info:
+      - name: Get info
+        community.aws.aws_s3_bucket_info:
```

Comment:
This doesn't work like that yet.

Also note, while setting the action FQCN in module_defaults will let the task use it if it's called with the same FQCN, redirected names don't work yet. The module_defaults action has to be an exact match with how the module is called. IMO it seems a little premature to collectionify the documentation until those things work.

Reply:
@s-hertel thanks for the feedback! I'll take a look at this tomorrow